opal-core-api/src/main/java/org/obiba/opal/core/domain/Project.java

```diff
@@ -53,6 +49,8 @@
 
   private String vcfStoreService;
 
+  private VCFSamplesMapping vcfSamplesMapping;
```

Comment:
I do not like the idea of having this data inside of the project document because project's edition is usually restricted to advanced users (administrator) and therefore should only contain configuration information. VCF samples mapping is more about data then you should persist this information in a new type of document, associated to the project by it's name. The project dto as well should not be changed.